### ISSUE TITLE
Is this repository unmaintained?

### ISSUE THREAD

**Issue by 1oglop1** [awslabs/aws-glue-libs#51]
Hi it looks like that this repository cannot keep up to date with the latest version of Glue, this makes it very difficult to implement tests and new functionality, by far AWS Glue looks like a most incomplete AWS product I've come across :(.

There are PRs waiting for review, issues are piling up.
Commits are made by people without associated Github account and it's not possible to tag them.
Also there are only 2 contributors while last commits were made by someone else. (Gh account issues)

Plus it looks really disconnected from https://github.com/aws-samples/aws-glue-samples

@jpeddicord 
@hyandell 
@moomindani

**Comment by itamar-otonomo** [comment]
It's possible this repo still exists only for legal reasons.

**Comment by MarcosBernal** [comment]
It seems so, but some clarification from AWS team will be appreciated!!!

**Comment by phtechwohletz** [comment]
Agreed, an update would be nice for the dev teams relying on this library.

**Comment by ismailsimsek** [comment]
Agree, an update would be nice
